documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,324 @@
+// * Link:https://docs.google.com/presentation/d/19H3tSzZ1pSGGwhSoZn3CFgyLQcBWbePAK0_5J4NHUGw/edit#slide=id.g22dc74ad918_0_740[Red Hat Secure Development Lifecycle Planning and Schedule]
+// * link:https://docs.google.com/presentation/d/1DOxSd5hpwNntypX5DUd3JRmP8wIJL_RVxOJfDo0Nxck/edit#slide=id.g13028f60288_0_0[Secure Development - Introduction to SSML]
+// * link:https://gitlab.cee.redhat.com/users/auth/geo/sign_in[Closed link]
+// * link:https://docs.engineering.redhat.com/display/PRODSEC/Secure+Development+training[Secure Development Training]
+
+.OCP Provider
+
+This is a collection of new features for {ocp-name} (OCP) provider. link:https://issues.redhat.com/browse/MTV-739[(MTV-739)]
+// 
+// .. Block Migrations from OpenShift versions before 4.13
+// 
+// This does not look like a feature. Is this a resolved issue?
+
+.. Secure connections to remote OpenShift clusters
+
+Connections to remote OpenShift clusters can be secured with CA certificates. link:https://issues.redhat.com/browse/MTV-728[MTV-728]
+
+// Is this a feature or Resolved issue?
+// .. Source and storage endpoints are not selected automatically
+// 
+// Source network and source storage anre not seleted automatically when creating an OCP to OCP plan. link:https://issues.redhat.com/browse/MTV-729[MTV729] 
+
+.Console Plugin
+
+This is a collection of new features for the {project-short} Console plugin. link:https://issues.redhat.com/browse/MTV-736[(MTV-736)]
+
+..MTV-615
+
+..MTV-737
+
+..MTV-245
+
+..MTV-365
+
+..MTV-449
+
+..MTV-616
+
+..MTV-773
+
+..MTV-724
+
+..MTV-766
+
+..MTV-768
+
+..MTV-770
+
+..MTV-772
+
+..MTV-775
+
+..MTV-779
+
+..MTV-792
+
+..MTV-793
+
+..MTV-797
+
+..MTV-867
+
+.OVA Provider
+
+OVA provider. link:https://issues.redhat.com/browse/MTV-690[(MTV-690)]
+
+..MTV-759
+
+..MTV-848
+
+..MTV-624
+
+..MTV-681
+
+..MTV-680
+
+..MTV-689
+
+..MTV-733
+
+..MTV-669
+
+..MTV-715
+
+..MTV-714
+
+..MTV-696
+
+[id="known-issues-26_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+.Deleting migration plan does not remove temporary resources
+
+Deleting a migration plan does not remove temporary resources such as importer pods, conversion pods, config maps, secrets, failed VMs, and data volumes. You must archive a migration plan before deleting it to clean up the temporary resources. link:https://bugzilla.redhat.com/show_bug.cgi?id=2018974[(BZ#2018974)]
+
+.Unclear error status message for VM with no operating system
+
+The error status message for a VM with no operating system on the *Plans* page of the web console does not describe the reason for the failure. link:https://bugzilla.redhat.com/show_bug.cgi?id=2008846[(BZ#22008846)]
+
+.Migration of virtual machines with encrypted partitions fails during conversion
+
+vSphere only: Migrations from {rhv-short} and OpenStack do not fail, but the encryption key might be missing on the target {ocp} cluster.
```

Comment:
```suggestion
vSphere only: Migrations from {rhv-short} and {osp} do not fail, but the encryption key might be missing on the target {ocp} cluster.
```

Could be written in a better way

Reply:
Now I'm thinking about the `vSphere only:` part..... Shall this be mentioned in the release note's title instead? 

Also, as I mentioned above, if there's no workaround for that, I'd recommend saying it explicitly.